GliaWidgets/Theme/Alert/SingleActionAlertConfiguration.swift

```diff
@@ -0,0 +1,11 @@
+/// Configuration of the operator ending the engagement alert.
+public struct SingleActionAlertConfiguration {
+    /// Title of the alert.
+    public var title: String?
```

Comment:
I'd like to clarify, do we need to have properties as `var` instead of `let`?
If we don't expect changing of alert configuration in runtime, let's make it as `let`.